gnovm/pkg/gnomod/gnomod_test.go

```diff
@@ -0,0 +1,136 @@
+package gnomod
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/gnolang/gno/tm2/pkg/testutils"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestCreateGnoModFile(t *testing.T) {
```

Comment:
Can you paralellize this test with `t.Parallel()`?

Reply:
Opps. 

Fixed here: 58f6ffa83d211acb9afa681f5d7cf1507280bdc3

Reply:
done efc2e5baea9877685b84b3b8d21e9bf5884e6a56